scala.html.markdown

```diff
@@ -614,22 +615,31 @@ List(
   Person(name = "Bob", age = 30)
 ).filter(_.age > 25) // List(Person("Bob", 30))
 
+// The reduce function takes a function of type A -> B -> A which it repeatedly
+// applies to the sequence, accumulating the result
+List(1, 2, 3) reduce (_ * _) // 6
+List(
+  "Dom",
+  "Bob",
+  "Alice"
+).reduce(_ + " " + _) // Dom Bob Alice
 
 // Certain collections (such as List) in Scala have a `foreach` method,
 // which takes as an argument a type returning Unit - that is, a void method
+// Some languages call this the `iter` method
```

Comment:
foreach functions are common enough that I don't think you need `iter` to illustrate. Most modern languages use forEach anyway. Please remove this line.